parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -58,12 +58,12 @@ setPWExpiration() {
   grep 'PASS_MIN_DAYS' /etc/login.defs && exit $ERR_CIS_APPLY_PASSWORD_CONFIG
   sed -i "s|INACTIVE=||g" /etc/default/useradd || exit $ERR_CIS_APPLY_PASSWORD_CONFIG
   grep 'INACTIVE=' /etc/default/useradd && exit $ERR_CIS_APPLY_PASSWORD_CONFIG
-  echo 'PASS_MAX_DAYS 90' >> /etc/login.defs || exit $ERR_CIS_APPLY_PASSWORD_CONFIG
-  grep 'PASS_MAX_DAYS 90' /etc/login.defs || exit $ERR_CIS_APPLY_PASSWORD_CONFIG
+  echo 'PASS_MAX_DAYS 99999' >> /etc/login.defs || exit $ERR_CIS_APPLY_PASSWORD_CONFIG
```

Comment:
Strictly speaking this change will expire passwords on vms sometime in 2293 or 2294, depending on when this PR merges. Are we assuming that climate change-induced geographical events will predict that Azure data centers won't remain viable in their current locales well before that time, and thus mooting any long-term eventual password expiration?

Reply:
That is exactly what we're predicting. Don't sell your MSFT stock though...at least until 2280.